### PR TITLE
Increase test coverage

### LINKED ISSUE #2: Increase the test coverage in Yaapii.Xml
### Bug Report or Feature Request (mark with an x)
- [ ] bug report -> please search issues before submitting
- [x] feature request
- [ ] refactor request

### Expected Behavior
The functions are tested

### Actual Behavior
Not all functions are tested

### PR DESCRIPTION
**Pull request by aveXcaesar**
### Please check if the PR fulfills these requirements
- [x] I made sure that my code builds (VisualStudio & cake script: build.ps1)
- [x] I merged the master into this branch before pushing
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe: Tests

### What is the current behavior? (You can also link to an open issue here)
closes #2 

### What is the new behavior?
Moore tests

### Does this PR introduce a breaking change? (check one with "x")
- [ ] Yes
- [x] No

###  Other information
closes #2

### CODE REVIEW

**Comment by aveXcaesar** [comment]
@Meerownymous 
Merged with master

File: tests/Test.Yaapii.Xml/TidyFileList.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Yaapii.Atoms;
+using Yaapii.Atoms.IO;
+using Yaapii.Atoms.Scalar;
+
+namespace Yaapii.Xml
+{
+    public sealed class TidyFileList : IAction
```
Comment (Meerownymous) [inline tests/Test.Yaapii.Xml/TidyFileList.cs:10]: missing header docs

File: tests/Test.Yaapii.Xml/TidyFileList.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        /// <param name="files"></param>
+        /// <param name="action"></param>
+        public TidyFileList(IDictionary<string, string> files, Action action) : this(
+            new ScalarOf<string>(() =>
+                Path.Combine(Directory.GetCurrentDirectory(), "TmpFileList")),
+            files,
+            action)
+        { }
+
+        public TidyFileList(string root, IDictionary<string, string> files, Action action) : this(
```
Comment (Meerownymous) [inline tests/Test.Yaapii.Xml/TidyFileList.cs:30]: missing header docs

File: tests/Test.Yaapii.Xml/XSLEnvelopeImplementation.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Xml;
+using Yaapii.Atoms;
+using Yaapii.Xml;
+
+namespace Yaapii.Xml
+{
+    public sealed class XSLEnvelopeImplementation : XSLEnvelope
```
Comment (Meerownymous) [inline tests/Test.Yaapii.Xml/XSLEnvelopeImplementation.cs:8]: missing header docs

**Review by Meerownymous — CHANGES_REQUESTED** [review]

**Pull request merged**